contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.0.0--3.1.0.sql

```diff
@@ -31,6 +31,26 @@ end
 $$
 LANGUAGE plpgsql;
 
+CREATE OR REPLACE FUNCTION sys.babelfish_update_server_collation_name() RETURNS VOID
```

Comment:
if this issue only comes up when we upgrade from 15.x (x<=6) to 17 latest then how is this upgrade path coming into picture?

Reply:
It is possible that when source is 15.x (x <6) and targrt is 17.y, then we will come till 15.6 and then upgrade to 16 and then 17. For the first step, it is possible that the collation of some view may have been changed. As our upgrade test framework does not cover all such possible upgrade paths for non-default collations, added this to be on the safe side.

Reply:
Got it